CacheService/docker_context/LegacyStockServerHandler.py

```diff
@@ -17,6 +25,28 @@ async def quoteSocket(self):
         return reader, writer
 
     async def getQuote(self, trans_num, user_id, stock_symbol):
+        if stock_symbol in self.tracker.keys():
+            runningTask = self.tracker[stock_symbol]
+            while not runningTask.done():
+                sleep(1)
```

Comment:
This is a blocking call, use `await asyncio.sleep`

Reply:
done.